mycore-orcid2/src/main/resources/xsl/orcid2/v3/mcr2work_generic.xsl

```diff
@@ -84,7 +86,14 @@
 
   <xsl:template match="mods:abstract">
     <work:short-description>
-      <xsl:value-of select="text()" />
+      <xsl:choose>
```

Comment:
should make use of `mcrstring:shorten(string, int)` (defined in `xslt/functions/stringutils.xsl`, namespace: `http://www.mycore.de/xslt/stringutils`) instead of reinventing the wheel. That method, for example, takes better care of placing the ellipsis at word eindings.